extensions/smallrye-graphql/deployment/src/main/java/io/quarkus/smallrye/graphql/deployment/SmallRyeGraphQLProcessor.java

```diff
@@ -363,6 +362,34 @@ private boolean getBooleanConfigValue(Optional<Boolean> quarkusConfig, String sm
         return classes;
     }
 
+    // Other Config Mappings
+
+    @BuildStep
+    void configMapping(SmallRyeGraphQLConfig graphQLConfig,
+            BuildProducer<SystemPropertyBuildItem> systemProperties) {
+
+        if (graphQLConfig.errorExtensionFields.isPresent()) {
+            systemProperties.produce(new SystemPropertyBuildItem(ConfigKey.ERROR_EXTENSION_FIELDS,
```

Comment:
Maybe we could detect that there is a `quarkus.*` property and the equivalent SmallRye/MP property at the same time, and if they have different values, throw a warning or something, because it's not clear which one should be used.

But that's just a corner case idea, I think this "quarkus-to-library" config mapping needs some more thinking in general, and perhaps a unified approach.

Reply:
I think the config folk are looking at something like this ...